tsconfig.base.json

```diff
@@ -45,7 +45,8 @@
       "@fluentui/make-styles-webpack-loader": ["packages/make-styles-webpack-loader/src/index.ts"],
       "@fluentui/react-avatar": ["packages/react-avatar/src/index.ts"],
       "@fluentui/react-card": ["packages/react-card/src/index.ts"],
-      "@fluentui/utilities": ["packages/utilities/src/index.ts"]
+      "@fluentui/utilities": ["packages/utilities/src/index.ts"],
```

Comment:
@Hotell  Question: Should this package @fluentui/utilities be removed from here? Because I think it is related to v8, and noticed that is was added in this PR: #19036

Reply:
it definitely should. This was done because how the generator works. it adds all migrated package dependencies to this base.json if not present. 

It is lacking deep dependencies check if those are truly vNext. 

Please remove this one.

great catch and thank you!

Reply:
I'll fix this in another PR, thank you 🙏

Reply:
sure thing. just a heads up, if you could create that PR today that would be great 🙌, as otherwise things may break (its already in master so.... :D )